oaknational/Oak-Web-Application
# feat: add placeholder theme colours from branding doc

## Description

Adds the colours in the brand document in a placeholder theme.


## How to test

1. Go to [storybook](https://samara-storybook-git-feat-placehold-d58a44-oak-national-academy.vercel.app/)
2. Click on button
3. Change them to "placeholder"
3. You should see button with green background.

## Screenshots
<img width="110" alt="Screenshot 2022-06-16 at 13 51 58" src="https://user-images.githubusercontent.com/2959739/174073720-45d3d508-91f0-4813-9c4a-fa61a1d764d7.png">

<img width="324" alt="Screenshot 2022-06-16 at 11 30 05" src="https://user-images.githubusercontent.com/2959739/174073759-60546e4b-2541-41ec-8b3f-9c1efc7a0c0a.png">


## Checklist
